_posts/2018-06-01-another-successful-rails-girls-event.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Another successful Rails Girls event"
+date: 2018-06-01 09:00:00
+categories: ["learning", "rails"]
+author: "luciano"
+---
+
+Last month was the fifth edition of the [Rails Girls Buenos Aires](http://railsgirls.com/buenosaires.html), and third participation by Ombu Labs as a sponsor and part of the coaching team. This time it took place in the impressive offices of Google Argentina where 25 participants attended. All of them were very excited and full of motivation to start learning new skills.
```

Comment:
It might be better to link to the English version of the local event instead: http://railsgirls.com/buenosaires-en.html